spec/Vehicle/DigitalKey.vspec

```diff
@@ -0,0 +1,236 @@
+
+DigitalKey.KeyData:
+  type: branch
+  description: Data related to the key at the index specified by DigitalKey.KeySelector.
+
+DigitalKey.KeyData.KeyState:
+  datatype: string
+  type: sensor
+  allowed: ['OUT_OF_RANGE', 'UNUSED', 'SETUP', 'NOT_ACTIVATED','ACTIVE', 'SUSPENDED', 'TERMINATING', 'TERMINATED', 'EXPIRED']
+  description: State for Digital Key at index given by DigitalKey.KeySelector.
+               OUT_OF_RANGE indicates that the vehicle does not store keys at the given index.
+               UNUSED indicates that this slot is UNUSED
+               SETUP indicates that key setup is in progress - key is not yet available.
+               NOT_ACTIVATED indicates that key setup is completed and the key is available for use,
+               but activation (if required) has not yet been performed.
+               ACTIVE indicates that the key setup is completed, the key is activated and valid for use. Note time period constraints below.
+               SUSPENDED indicates that key has been temporarily disabled.
+               TERMINATING indicates that process for terminating the key has been initiated
+               TERMINATED indicates that key has been terminated.
+               EXPIRED indicates that key has expired and is no longer valid.
+               In the states OUT_OF_RANGE, UNUSED and SETUP other signals of DigitalKey.KeyData shall not be used.
+
+DigitalKey.KeyData.RequestedState:
+  datatype: string
+  type: actuator
+  allowed: ['NONE', 'TERMINATE', 'SUSPEND', 'RESUME']
+  description: Specifies if there is a request to change state of the key.
+               NONE indicates that no request is active.
+               TERMINATE indicates a request to permanently terminate the key.
+               SUSPEND indicates a request to temporarily suspend the key.
+               RESUME indicates a request to resume key status from SUSPENDED to ACTIVE.
+  comment: This signal can be used by a client to initiate a request.
+           Additional steps not part of VSS may be required to actually change the state of key.
+
+DigitalKey.KeyData.GlobalKeyIdentifier:
+  type: sensor
+  datatype: uint8[]
+  description: Globally unique identifier of the key.
+  comment: Byte/char array used to uniquely identify this key.
+
+DigitalKey.KeyData.VehicleKeyIdentifier:
+  type: sensor
+  datatype: uint8[]
+  description: Vehicle unique identifier of the key.
+  comment: Byte/char array used to uniquely identify this key within the vehicle.
+           This may be identical to DigitalKey.KeyData.KeyIdentifier but could as well be something
+           different.
+
+DigitalKey.KeyData.KeyCapability:
+  type: sensor
+  datatype: string
+  description: String indicating capabilities of the key.
+               Allowed values and interpretation is implementation-specific.
+
+DigitalKey.KeyData.StartTime:
+  type: sensor
+  datatype: uint64
+  description: Start time of validity for the key.
+  unit: unix-time
```

Comment:
This should be UTC and not "wall clock" time. Same for other times herein.

Reply:
That is implied by our definition of `unix-time` in https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/units.yaml#L213:

`Number of non-leap seconds which have passed since 00:00:00 UTC on Thursday, 1 January 1970`